docs/source/developers/cpp/development.rst

```diff
@@ -85,6 +85,8 @@ This project follows `Google's C++ Style Guide
   conveniences.  For private headers the rules are relaxed further and structs
   can be used where convenient for types that do not need access control even
   though they may not be simple data containers.
+* We prefer pointers for output and input/output parameters (the
+  style guide recommends mutable references in some cases).
```

Comment:
Thanks for this. I would remove "minor" above from the "minor exceptions" wording since this is a significant deviation.

Reply:
Good point.  I changed "minor" to "these".